frontend/src/pages/main/payment/components/confirm/PaymentConfirmation.tsx

```diff
@@ -76,9 +74,8 @@ const PaymentConfirmation: React.FC<ConfirmProps> = ({qrCodeData}) => {
             <CardContent className="card-confirm-content">
                 <TextContent text={Content.INFORMATION}/>
                 <BalanceField/>
```

Comment:
You are getting wallet in PaymentConfirmation and in balance field. On the same page 2 same requests. Bad.
Instead of `<BalanceField/>` use `<FieldConfirmation>` and pass balance and prefix which you have